packages/app/src/settings/device/NotificationSettings.vue

```diff
@@ -6,77 +6,104 @@
     <template #description>
       {{ t('settingsPage.notifications.description') }}
     </template>
-    <div class="divide-y border rounded divide-gray-50 border-gray-100 px-[16px]">
+    <div class="divide-y border rounded divide-gray-50 border-gray-100">
       <div
-        v-for="({id, title}) in switches"
-        :key="id"
-        class="py-[16px]"
+        v-if="!props.gql.localSettings.preferences.desktopNotificationsEnabled"
+        class="bg-indigo-100 p-[16px] flex justify-between items-center"
+        data-cy="enable-notifications"
       >
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          <label :for="id">{{ title }}</label>
-          <Switch
-            class="mx-[8px]"
-            :value="props.gql.localSettings?.preferences[id] ?? false"
-            :name="id"
-            @update="(value) => updatePref(value, id)"
-          />
-          {{ gql.localSettings }}
-        </h4>
-      </div>
-      <div class="py-[16px]">
-        <h4 class="flex text-gray-800 text-[16px] leading-[24px] items-center">
-          {{ t('settingsPage.notifications.notifyMeWhenRunCompletes') }}
-        </h4>
-        <div class="pt-[5px]">
-          <Checkbox
-            v-for="({id, label}) in statuses"
-            :id="id"
-            :key="id"
-            v-model="listRef"
-          >
-            <template #label>
-              <label
-                :for="id"
-                class="text-sm text-gray-900"
-              >
-                {{ label }}
-              </label>
-            </template>
-          </Checkbox>
+        <div class="text-indigo-700 font-medium">
```

Comment:
In Figma I see a padlock icon here, but I don't see it in the app

<img width="163" alt="Screenshot 2023-06-12 at 10 09 16 AM" src="https://github.com/cypress-io/cypress/assets/14275198/de49cce5-9947-4d5e-8032-a26d3c1bdf3d">